### DISCUSSION TITLE
sidebar vanished completely

### DISCUSSION THREAD

**Discussion by sickbvbblegum** [AprilSylph/XKit-Rewritten#237] [Help Desk] [answered]
(sorry if this is a weird format i've never used github before) I had the no recommended option enabled, but when i disabled it my sidebar is still gone. No recommended blogs or tumblr radar, regardless of how many times i refresh/ click around.

**Comment by AprilSylph** [comment, marked as answer]
Do you have Tumblr Savior installed, by any chance?

**Reply by sickbvbblegum** [reply]
i do, actually. just looked at the settings on tumblr savior (downloaded it today) and that's the issue. thank you!
